### DISCUSSION TITLE
Full Duplex or Half Duplex?

### DISCUSSION THREAD

**Discussion by vtjballeng** [will127534/LoRa-GW-FeatherWing#2] [Ideas]
@will127534 are there any plans to support the LoRa corecell reference design Full Duplex capability either in this GW or your LoRa concentrator?

**Comment by will127534** [comment]
Probably Not?
Full Duplex capability looks like you need to attach two antenna, which is not something I want to...

Definitely not do-able on Featherwing format, there is no place to place another SMA port.

The closest thing is the Version 2 of the LoRa-concentrator, the only different is that I didn't put any PA or LNA on board and the SX1261 for LBT.  
https://github.com/will127534/LoRa-concentrator/
